res/i18n/strings_zh-cn.i18n.json

```diff
@@ -0,0 +1,640 @@
+    "everyoneAverages": "Tetra联赛散点图",
+    "sortBy": "排序依据",
+    "reversed": "反向",
+    "country": "地区",
+    "rankAverages": "$rank段位散点图",
+    "players": {
+        "zero": "$n 个玩家",
+        "one": "$n 个玩家",
+        "two": "$n 个玩家",
+        "few": "$n 个玩家",
+        "many": "$n 个玩家",
+        "other": "$n 个玩家"
+    },
+    "games": {
+        "zero": "$n 场游戏",
+        "one": "$n 场游戏",
+        "two": "$n 场游戏",
+        "few": "$n 场游戏",
+        "many": "$n 场游戏",
+        "other": "$n 场游戏"
+    },
+    "gamesPlayed": "$games 场游戏",
+    "chart": "列表",
+    "entries": "条目",
+    "minimums": "最小值",
+    "maximums": "最大值",
+    "lowestValues": "最小值",
+    "averageValues": "平均值",
+    "highestValues": "最大值",
+    "forPlayer": "来自用户 $username",
+    "currentAxis": "$axis 轴：",
+    "p1nkl0bst3rAlert": "该数据是从 p1nkl0bst3r 维护的第三方 API 中检索的",
+    "notForWeb": "浏览器版本暂不支持函数",
+    "graphs": {
+        "attack": "攻击",
+        "speed": "速度",
+        "defense": "防御",
+        "cheese": "奶酪层"
+    },
+    "statCellNum": {
+        "xpLevel": "XP等级",
+        "xpProgress": "到下一等级的进度",
+        "xpFrom0ToLevel": "从 0 到 $n 等级的进度",
+        "xpLeft": "XP 还有",
+        "hoursPlayed": "小时游玩",
+        "onlineGames": "在线游戏场次",
+        "gamesWon": "获胜场次",
+        "totalGames": "总在线游戏场次",
+        "totalWon": "总在线游戏获胜场次",
+        "friends": "好友",
+        "apm": "每分\n发送垃圾行",
+        "vs": "VS\n分数",
+        "recordLB": "名次",
+        "lbp": "名次",
+        "lbpShort": "名次",
+        "lbpc": "地区\n名次",
+        "lbpcShort": "地区名次",
+        "gamesPlayed": "游戏\n场次",
+        "gamesWonTL": "获胜\n场次",
+        "winrate": "胜率",
+        "level": "等级",
+        "score": "分数",
+        "spp": "每块\n得分",
+        "pieces": "放置\n块数",
+        "pps": "每秒\n放置块数",
+        "finesseFaults": "非极简\n操作",
+        "finessePercentage": "极简率",
+        "keys": "按键",
+        "kpp": "每块\n按键",
+        "kps": "每秒\n按键",
+        "tr": "Tetra分数",
+        "rd": "偏移值",
+        "app": "每块发送垃圾行数",
+        "appDescription": "(Attack per Piece, 简称APP) 主要效率指标。表示玩家每块可以发动多少次攻击",
+        "vsapmDescription": "基本上可以告诉你在攻击中利用垃圾行的效率",
+        "dss": "每秒\n挖掘",
+        "dssDescription": "(Downstack per Second, 简称 DS/S) 测量一秒钟内清除多少条垃圾行。",
+        "dsp": "每块\n挖掘",
+        "dspDescription": "(Downstack per Piece, 简称 DS/P) 测量每一块清除多少条垃圾行。",
+        "appdsp": "APP + DS/P",
+        "appdspDescription": "只是每块发送垃圾行数与每块挖掘之和。",
+        "cheese": "垃圾行\n混乱指数",
+        "cheeseDescription": "(Cheese Index, 简称Cheese) 是玩家发出的垃圾行有多整齐/混乱的近似值。低数据代表整齐，高数据代表混乱。\n由 kerrmunism 发明",
+        "gbe": "垃圾行\n效率",
+        "gbeDescription": "(Garbage Efficity, 简称Gb Eff.) 测量玩家如何很好地利用他们收到的垃圾行。高数据代表更好或者他们更多地用TA的垃圾行，低数据代表TA大多将垃圾行送回奶酪层，或者很少清理垃圾行。\n由 Zepheniah 与 Dragonboy 发明。",
+        "nyaapp": "加权\nAPP",
+        "nyaappDescription": "(Weighted APP, 简称wAPP) 在本质上是在衡量您在保持高 APP 的同时发送奶酪的能力。\n由 Wertj 发明。",
+        "area": "面积",
+        "areaDescription": 如果排除 Cheese 和 vs/apm 部分，您的形状在图表上占据了多少空间",
```

Comment:
You forgot to add "

Reply:
Fixed.
Btw, thank you for accepting my translation!